language/tools/move-cli/README.md

```diff
@@ -35,8 +35,27 @@ you can find the complete list of commands available by calling `move
 for each Move CLI command can be found by passing the `--help` flag to it,
 i.e., `move <command> --help`.
 
+Move CLI commands are grouped into three large subcommands:
+* **base commands**: are basic Move commands such as compiling or checking a set of modules. These do not rely on a Move Adapter implementation nor implementation of storage.
+* **sandbox commands**: are commands that allow you to write Move modules and scripts, then run scripts and tests, and view the resulting state of execution in a local sandboxed environment.
+* **experimental commands**: are experimental commands that are currently in development.
 
-## Project structure
+## Base Commands
+
+You can use the `move check` command to compile and typecheck either a specific set of
+modules, or all modules under the passed-in directory:
+
+```shell
+$ move check <move_file_1> ... <move_file_n>
+$ move check <dir_1> ... <dir_n>
+```
+
+This will cause the CLI to compile and typecheck the provided module(s), but it won't
+publish the module bytecode, nor will it output the compiled bytecode.
+
+## Sandbox Commands
```

Comment:
> Persistent data is stored on-disk in a directory structure that
mimics the Move memory model

This text appears at the beginning of the file, but might be nice to move a massaged version of it here to make it clear that this is the "sandbox"